<!-- Make sure launch ctrl_preapare first -->
<launch>
    <param name="/use_sim_time" value="true" />
    <node pkg="thewalkingdead" type="task11.py" name="task11" output="screen"/>
</launch>